hysr/hysr_control.py

```diff
@@ -1,3 +1,8 @@
+import pathlib
+import toml
```

Comment:
I added in this pull request in pyproject.toml the dependency on types-toml, so these warnings should go away one the merge is done

Reply:
I'm afraid, this won't help.  Currently the mypy check is not installing the package, thus ignoring any dependencies.
Not sure if there is a good generic solution because installing the package for the check would not be so trivial for CMake packages that also install some Python code.
`types-PyYAML` already gets installed explicitly [here](https://github.com/intelligent-soft-robots/hysr/blob/8583107b446fb7352506d5eea780b90b37b1880f/.github/workflows/python_mypy.yml#L15), maybe we can just increase this list?

Btw: If it would help, it should already pass here, not only after merging.